### PR TITLE
AQCU-1644: create Dockerfile for repgen

### PR DESCRIPTION
**Pull request by mhines-usgs**

### CODE REVIEW

**Comment by isuftin** [comment]
Do you have a link to the base container repo

**Comment by mhines-usgs** [comment]
@isuftin https://github.com/USGS-CIDA/docker-aqcu-rserve/pull/22

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+sudo: required
+
+env:
+  global:
+    - DOCKER_COMPOSE_VERSION=1.21.1
```
Comment (isuftin) [inline .travis.yml]: Update to 1.22.0

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  - chmod +x docker-compose
+  - sudo mv docker-compose /usr/local/bin
+  - curl -L https://github.com/arminc/clair-scanner/releases/download/v8/clair-scanner_linux_amd64 > clair-scanner
+  - chmod +x clair-scanner
+  - sudo mv clair-scanner /usr/local/bin
+  - sudo apt-get -y -o Dpkg::Options::="--force-confnew" install docker-ce
+  - git clone https://github.com/docker/docker-bench-security.git
+
+install:
+  - docker build -t $DOCKER_IMAGE .
```
Comment (isuftin) [inline .travis.yml]: Remove this

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+version: '3.4'
+
+
+services:
+  repgen:
+    build: .
+    image: aqcu-repgen:0.0.1
```
Comment (isuftin) [inline docker-compose.yml]: Are you sure you want to version constrain here?

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+---
+version: '3.4'
```
Comment (isuftin) [inline docker-compose.yml]: Go to 3.5

File: .travis.yml [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+  - docker-compose -f docker-compose.yml up -d --build
+
+script:
+  - docker ps | grep -q repgen
+  - cd docker-bench-security
+  - sudo sh docker-bench-security.sh -c container_images
+  - docker run -d --name db arminc/clair-db
+  - docker run -p 6060:6060 --link db:postgres -d --name clair arminc/clair-local-scan:v2.0.3
```
Comment (isuftin) [inline .travis.yml]: Change to `v2.0.4` at the end there

File: docker-compose.yml (hunk 3 above)
Comment (mhines-usgs) [reply docker-compose.yml]: not necessarily, do i just want latest then? I wasn't sure what was appropriate

Comment (isuftin) [reply docker-compose.yml]: I'd just suggest latest for the time being

File: .travis.yml [hunk 6]
```diff
@@ -0,0 +1,32 @@
+sudo: required
+
+env:
+  global:
+    - DOCKER_COMPOSE_VERSION=1.22
```
Comment (isuftin) [inline .travis.yml]: 1.22.0

**Comment by isuftin** [comment]
👍

**Pull request merged**